### PR TITLE
testing/twine : new aport 

### PR DESCRIPTION
**Pull request by neofutur**
testing/py-twine : new aport  
5 commits,for twine and the 4 other apkbuilds necessary dependencies ( py-bleach py-pkginfo py-readme_renderer and py-webencodings )

### CODE REVIEW

File: testing/py-bleach/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+pkgrel=3
+pkgdesc="Bleach is an allowed-list-based HTML sanitizing library that escapes or strips markup and attributes."
+url="https://pypi.org/project/bleach/"
+arch="noarch"
+license="Apache-2.0"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools py3-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+#source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+source="https://files.pythonhosted.org/packages/ae/31/680afc7d44040004296a2d8f0584983c2f2386448cd9d0964197e6c1160e/bleach-3.0.2.tar.gz"
```
Comment (andypost) [inline testing/py-bleach/APKBUILD]: needs fix

File: testing/py-bleach/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+package() {
```
Comment (andypost) [inline testing/py-bleach/APKBUILD]: there's no tests?

Comment (andypost) [reply testing/py-bleach/APKBUILD]: I mean needs `check()`

File: testing/py-pkginfo/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+pkgrel=2
+pkgdesc="This package provides an API for querying the distutils metadata written in the PKG-INFO file inside a source distriubtion"
+url="https://pypi.org/project/pkginfo/"
+arch="noarch"
+license="Apache-2.0"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools py3-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+#source="https://files.pythonhosted.org/packages/39/c9/111ececbeed8e69cd1b6bec79a32a0b0f6074038a4244e58e285ad278248/pkginfo-1.4.2.tar.gz"
```
Comment (andypost) [inline testing/py-pkginfo/APKBUILD]: needs fix and also why no `check()` implemented?

File: testing/py-bleach/APKBUILD (hunk 1 above)
Comment (neofutur) [reply testing/py-bleach/APKBUILD]: I have no idea about wich tests you could want :(  I fixed the non normalized source url )

File: testing/py-pkginfo/APKBUILD (hunk 3 above)
Comment (neofutur) [reply testing/py-pkginfo/APKBUILD]: I have no idea what test I should put in check,or where to find them. 
having a commented line with # is a problem ?

**Pull request closed without merging**